README.md

```diff
@@ -106,18 +106,18 @@ A persona tag is essentially a cardinal based user. To create a persona tag in y
 endpoint from the dropdown. Make sure to paste in a valid User ID into the User ID field. Set the request body to:
 ```json
 {
-  "persona_tag": "some-persona-tag"
+  "personaTag": "some-persona-tag"
 }
 ```
 
 and hit Submit. You should see a response like:
 
 ```json
 {
-  "persona_tag": "some-persona-tag",
+  "personaTag": "some-persona-tag",
   "status": "pending",
   "tick": 2567,
-  "tx_hash": "0x6bc26694dee4c4163335e4fe01d73eab2da071f38b991ae8424fa52de330c228"
+  "hash": "0x6bc26694dee4c4163335e4fe01d73eab2da071f38b991ae8424fa52de330c228"
```

Comment:
this should be txHash to match the change down in `nakama/persona_tag_storage.go`